DllImportGenerator/DllImportGenerator/Marshalling/MarshallingGenerator.cs

```diff
@@ -147,6 +147,14 @@ internal class MarshallingGenerators
                     or { ManagedType: { SpecialType: SpecialType.System_Double }, MarshallingAttributeInfo: NoMarshallingInfo or MarshalAsInfo(UnmanagedType.R8, _) }:
                     return Blittable;
 
+                // Enum with no marshalling info
+                case { ManagedType: { TypeKind: TypeKind.Enum }, MarshallingAttributeInfo: NoMarshallingInfo }:
+                    return Blittable;
+
+                // Pointer with no marshalling info
+                case { ManagedType: { TypeKind: TypeKind.Pointer }, MarshallingAttributeInfo: NoMarshallingInfo }:
```

Comment:
Do we want to let through pointers to non-blittable types or just pointers to blittable types here?

Reply:
We allow any pointers to go through in the built-in interop. I think we should do the same here.